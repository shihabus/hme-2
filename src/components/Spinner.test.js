import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Spinner from "./Spinner";

test("Spinner snapshot", () => {
  const { container } = render(<Spinner />);
  expect(container.firstChild).toMatchSnapshot();
});
