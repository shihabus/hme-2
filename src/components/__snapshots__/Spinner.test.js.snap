// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snapshot 1`] = `
<div
  class="sc-AxjAm ea-dyAp"
>
  <div
    class="sc-AxirZ iwZZzd"
  />
  <p
    class="sc-AxiKw gSxgiO"
  />
</div>
`;
